### DISCUSSION TITLE
Get Havok games running (Legend of Zelda;BOTW,Saints Row,Assasins creed etc)

### DISCUSSION THREAD

**Discussion by Xston02** [skyline-emu/skyline#615] [Game Requests]
Huge amount of games run on havok based engine on switch ,most of them being proper AAA titles..would love to see them working on skyline

**Comment by PixelyIon** [comment]
Havok is middleware for physics, there's massive variance between engines using them. Create requests based off individual titles or very similar ones (which should be on the same engine) rather than catch-all requests.
